be/src/exec/schema_scanner/schema_materialized_views_scanner.cpp

```diff
@@ -28,12 +28,20 @@ SchemaScanner::ColumnDesc SchemaMaterializedViewsScanner::_s_tbls_columns[] = {
         {"TABLE_NAME", TYPE_VARCHAR, sizeof(StringValue), false},
         {"REFRESH_TYPE", TYPE_VARCHAR, sizeof(StringValue), false},
         {"IS_ACTIVE", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"partition_type", TYPE_VARCHAR, sizeof(StringValue), false},
         {"LAST_REFRESH_START_TIME", TYPE_VARCHAR, sizeof(StringValue), false},
         {"LAST_REFRESH_FINISHED_TIME", TYPE_VARCHAR, sizeof(StringValue), false},
         {"LAST_REFRESH_DURATION", TYPE_VARCHAR, sizeof(StringValue), false},
         {"LAST_REFRESH_STATE", TYPE_VARCHAR, sizeof(StringValue), false},
-        {"INACTIVE_CODE", TYPE_VARCHAR, sizeof(StringValue), false},
-        {"INACTIVE_REASON", TYPE_VARCHAR, sizeof(StringValue), false},
+
+        {"FORCE_REFRESH", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"START_PARTITION", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"END_PARTITION", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"BASE_REFRESH_PARTITIONS", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"MV_REFRESH_PARTITIONS", TYPE_VARCHAR, sizeof(StringValue), false},
```

Comment:
better to add refresh task id and name into show mv's result
so that we can get the mv's related refresh task info.

Reply:
added